cloud-common/src/main/java/com/sequenceiq/cloudbreak/EnvironmentVariableConfig.java

```diff
@@ -75,18 +75,18 @@
             .append(String.format("BRAZIL_SOUTH:https://sequenceiqbrazilsouth2.blob.core.windows.net/images/%s", CB_AZURE_IMAGE_NAME))
             .toString();
     public static final String CB_AWS_AMI_MAP = new StringBuilder()
-            .append("ap-northeast-1:ami-4ea78120,")
-            .append("ap-southeast-1:ami-babd7bd9,")
-            .append("ap-southeast-2:ami-9188d6f2,")
-            .append("eu-central-1:ami-1cedfe70,")
-            .append("eu-west-1:ami-24a47c57,")
-            .append("sa-east-1:ami-6408b308,")
-            .append("us-east-1:ami-2393ed49,")
-            .append("us-west-1:ami-4af49b2a,")
-            .append("us-west-2:ami-203c2a41")
+            .append("ap-northeast-1:ami-31466f5f,")
+            .append("ap-southeast-1:ami-11111111,")
```

Comment:
true, as for several tries amazon was always failing fo ap-xxx regions. So we aggreed with @akanto that, first move on.

I was testing this code change, but only with local built jar/docker image. We need official jar/docker (let say 1.1.0-rc.43) to do repetable tests.

Therefore i propose that we create a 1.1.0-rc.43 which excludes ap-xxx regions, and while we redo the tests we can create those missing amis.